numba_dppy/dpnp_glue/dpnpdecl.py

```diff
@@ -0,0 +1,10 @@
+from numba.core.typing.templates import (AttributeTemplate, infer, infer_getattr, AbstractTemplate, signature)
+import numba_dppy
+from numba import types
+
+@infer_getattr
+class DpplDpnpTemplate(AttributeTemplate):
```

Comment:
s/Dppl/Dppy/g we are migrating to new naming convention next release.

Reply:
Updated.